src/main/java/com/Barker/model/Dog.java

```diff
@@ -61,6 +64,16 @@
 	@Column(name = "is_adopted")
 	private boolean isAdopted;
 	
+	@JsonIgnore
```

Comment:
Careful using JsonIgnore, I believe spring boot relies on object mapper when it does certain "magical" things and  when I tried using it to ignore the password field from being returned on a response object it messed up certain getX methods.

Reply:
Yeah, I was getting an infinite loop so I added @JsonIgnore.  If we have problems later we might have to find a different solution.  I did add a getter for shelterId so we get the shelterId in the body of the response.  We can use this to keep track of which shelter the dog belongs to if needed...

Reply:
.DS_Store was removed.